src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -210,37 +210,23 @@ public override ImmutableArray<StatementSyntax> GetOuterReturnStatements(SyntaxN
         }
 
         public override bool IsFinalSpanSemanticallyValidSpan(
-            TextSpan textSpan, ImmutableArray<StatementSyntax> returnStatements, CancellationToken cancellationToken)
```

Comment:
very odd use of passing spans of selections around, when we have the actual data we need in nodes that we can use directly.